tests/manage/pv_services/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -44,37 +44,42 @@ def check_ceph_used_space():
         return used_in_gb
     used_space = used_in_gb
     raise UnexpectedBehaviour(
-        f"In Ceph status, used size is keeping varying"
+        f"In Ceph status, used size is varying"
     )
 
 
 @retry(UnexpectedBehaviour, tries=5, delay=3, backoff=1)
-def verify_pv_not_exists(pvc_obj, cbp_name):
+def verify_pv_not_exists(pvc_obj, cbp_name, rbd_image_id):
     """
     Ensure that pv does not exists
     """
 
     # Validate on ceph side
-    logger.info(f"Verifying pv {pvc_obj.backed_pv} exists on backend")
+    logger.info(f"Verifying PV {pvc_obj.backed_pv} exists on backend")
 
-    _rc = pvc_obj.verify_pv_exists_in_backend(cbp_name)
+    _rc = helpers.verify_volume_deleted_in_backend(
```

Comment:
usually rc (return code) implies if the function failed or succeeded with rc>=0 (failure) and rc=0 success.
This usage of _rc is confusing.
Could you please change the parameter name to 'status' or 'deleted'?

Reply:
ack